conda/environments/all_cuda-118_arch-x86_64.yaml

```
@@ -26,7 +26,7 @@ dependencies:
 - libtool
 - ninja
 - numba>=0.57.1
-- numpy>=1.23,<2.0a0
+- numpy>=1.23,<3
```

```suggestion
- numpy>=1.23,<3.0a0
```
I think you want to keep the a0 junk. If you don't, then pre-release versions of v3.0 will be allowed.

I could well be operating on inaccurate historical information, though.